main/src/mockit/internal/startup/Startup.java

```diff
@@ -180,6 +180,13 @@ private static Instrumentation getInstrumentation()
                reapplyStartupMocks();
             }
          }
+
+         Class<?> initialMockingBridgeClass = ClassLoad.loadClass(systemCL, MockingBridge.class.getName());
+
+         if ((initialMockingBridgeClass != null) && (instrumentation != null)) {
+            MockingBridge.setHostClassName(FieldReflection.<String>getField(initialMockingBridgeClass, "hostClassName", null));
+            FieldReflection.setField(MockingBridge.class, null, "fieldsSet", true);
```

Comment:
Hm, maybe this line was not the best change. It actually is the problem that arises now.
But I think there is a base problem. Without this, the mocking bridge fields get set to new instances from the custom classloader. This was also the case in 1.24.

With this line, the mocking bridge fields are only set once with instances from the system class loader, but then it fails at runtime on the custom classloader here, because `mockupClassesToMockupInstances` is empty in the class from the system class loader.

But without the line, the tests that run after my custom class loader test fail, because then it is the other way around, the test that run on the system class loader then have the instances from the wrong class loader.

Maybe those fields should not be simple fields, but a Map from class loader to mocking bridge field, similar to a `ThreadLocal`?